.circleci/config.yml

```diff
@@ -0,0 +1,116 @@
+version: 2.1
+orbs:
+  win: circleci/windows@2.2.0
+
+runOnAllTagsAndBranches: &runOnAllTagsAndBranches
+  filters:
+    tags:
+      only: /.*/
+    branches:
+      only: /.*/
+
+jobs:
+  style:
+    docker:
+      - image: circleci/golang:1.15
+    steps:
+      - checkout
+      - run: go vet ./...
+      - run: if [ $(go fmt ./... | wc -l) -eq 0  ]; then exit 0; else exit 0; fi
+
+  test:
+    docker:
+      - image: circleci/golang:1.15
+
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 19.03.13
+      - run:
+          name: Run tests
+          command: |
+            make test-coverage-all-dockerized-with-codeclimate
+
+  # higher support plan is necessary
+  # test-macos:
+  #   macos:
+  #     xcode: 11.7.0
+  #   steps:
+  #   - checkout
+  #   - run: go test ./...
+
+  test-windows:
+    executor:
+      name: win/default
+      shell: powershell.exe
+    steps:
+      - checkout
+      - run: choco install make
+      - run: choco install golang --version 1.15
+      - run: make test
+
+  integration:
+    docker:
+      - image: circleci/golang:1.15
+    steps:
+      - checkout
+      - setup_remote_docker:
+          version: 19.03.13
+      - run:
+          name: Install commander
+          command: |
+            mkdir -p ~/.local/bin/
+            curl -L https://github.com/commander-cli/commander/releases/download/v2.1.0/commander-linux-amd64 -o ~/.local/bin/commander
+            chmod +x ~/.local/bin/commander
+      - run: make integration-linux-dockerized
+
+  integration-windows:
```

Comment:
We could guarantee some coverage by using `powershell` on linux. We'd be losing the ability to test `cmd`. How often are we able to run windows tests with our current support tier? I see their free tier states `2,500 free credits/week`, but I can't find any information stating that windows is capped?

Reply:
Do we get the same outputs from it as on windows, i.e. path separators?
In general we don't test the shell, it is about the operating system commander runs on.

Reply:
I believe so, but i agree. I don't think it's really worth investigating